wpilibc/BUILD.bazel

```diff
@@ -289,7 +289,7 @@ define_pybind_library(
 )
 
 robotpy_py_test(
-    "python_tests",
+    "wpilibc-python-test",
```

Comment:
Should we drop the package name from the target name? Seems rather verbose. So we'd end up with like `bazel test //wpilibc:cpp-test //wpilibc:python-test`.